dss/dynamodb/__init__.py

```diff
@@ -0,0 +1,67 @@
+from dss.util.aws.clients import dynamodb as db  # type: ignore
```

Comment:
The methods in this class could use docstrings.

For instance, when must `sort_key` be passed in?